base_config.json

```diff
@@ -0,0 +1,8 @@
+{
+    "s3_region": null,
```

Comment:
is there a specific reason why this was changed all to small letters?
Maybe all can be consistent, either small or capital letters (note: this would require change on aws-sdk upload/download, in the previous iteration i had a bug because of an outdated version of s3 region - refinery ui was expecting capital letters)

Reply:
this was a mistake and should be the same as before

Reply:
kept with diverging names